### ISSUE TITLE
Causes issue with Chrome Inspector

### ISSUE THREAD

**Issue by tim92109** [LeaVerou/prefixfree#6096]
When using prefixfree, when you inspect an element with Chrome Inspector, you will no longer see exactly which line number of your style sheet that the class or ID is being called from.  Real pain to have to search your style sheet every time you want to edit an element.

**Comment by nlourenco** [comment]
Same issue happened with me and its not limited to Chrome. All external styles are moved inline.

**Comment by tim92109** [comment]
Yea... I loved it at first, but stopped using it as it ended up eating more time than saving time.... just from having to search for elements.

**Comment by LeaVerou** [comment]
Unfortunately, that's unavoidable at the moment with any JS solution that modifies CSS, as it has to re-request the stylesheet.
Perhaps sourcemaps could help, though I haven't seen any such use of them.

**Comment by tim92109** [comment]
Makes sense.  BTW, if you're only coding for modern browsers, is there still a use for prefixfree in 2016?

**Comment by LeaVerou** [comment]
Certainly not as much as in 2012 when I made it. I still use it, but rarely.
